.ruby-version

```diff
@@ -0,0 +1 @@
+2.6.8
```

Comment:
This got committed by accident, but we could choose to leave it here?

Reply:
Hrm, I guess I don't feel strongly one way or the other but maybe we can bump this to a newer version if we're going to commit it?

Reply:
I went ahead and removed it as we should be supporting a range of Ruby versions! 👍